core/templates/dev/head/pages/topic-editor-page/questions-tab/questions-tab.directive.ts

```diff
@@ -36,6 +36,7 @@ require(
 require('pages/topic-editor-page/services/topic-editor-state.service.ts');
 require('services/AlertsService.ts');
 require('services/contextual/UrlService.ts');
+require('services/QuestionsListService.ts');
```

Comment:
Switch line 38 and 39. We are ordering also by the depth of the folder structure where is the file located.

Reply:
Done.